contracts/contracts/Multisig.cairo

```diff
@@ -322,80 +311,65 @@ end
 #
 
 @constructor
-func constructor{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr,
-    }(
-        owners_len : felt,
-        owners : felt*,
-        confirmations_required : felt,
-    ):
-    with_attr error_message("invalid number of required confirmations"):
-        assert_le(confirmations_required, owners_len)
+func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    owners_len : felt, owners : felt*, confirmations_required : felt
+):
+    const lower_bound = 1
+
+    with_attr error_message("range conditions unsatisfied"):
+        assert_in_range(confirmations_required, lower_bound, owners_len + 1)
     end
 
-    _owners_len.write(value=owners_len)
-    _set_owners(owners_index=0, owners_len=owners_len, owners=owners)
-    _confirmations_required.write(value=confirmations_required)
+    _set_owners_impl(owners_len, owners)
```

Comment:
Well here I've added `_impl` suffix because `_set_owners` is already exists and used inside `_set_owners_impl`. I've mentioned that on slack that I could use some naming suggestions here. Don't like it either

Reply:
Let's rename like this:

- `_set_owners_impl` -> `_set_owners`
- `_set_owners` -> `_set_owners_range`
- `_clean_owners` -> `_clean_owners_range`